code/game/objects/items/toys.dm

```diff
@@ -123,6 +123,32 @@
 	user.visible_message("<span class='notice'>[user] plays with [src].</span>", "<span class='notice'>You [playverb].</span>")
 	lastused = world.time
 
+/obj/item/toy/syndicateballoon/suicide_act(mob/living/user)
+	. = ..()
+	if(!user)
+		return
+
+	user.visible_message("<span class='suicide'>[user] ties [src] around their neck and starts to float away! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+
+	// playsound(get_turf(user), 'sound/magic/tail_swing.ogg', 80, TRUE)
```

Comment:
```suggestion
```
Don't think you're supposed to leave commented-out code in

Reply:
true